src/framework/vst/internal/vstmodulesrepository.cpp

```
@@ -162,15 +166,19 @@ audio::AudioResourceMetaList VstModulesRepository::modulesMetaList(const VstPlug
     return result;
 }
 
-RetVal<io::paths> VstModulesRepository::pluginPathsFromCustomLocation(const io::path& customPath) const
+io::paths VstModulesRepository::pluginPathsFromCustomLocations(const io::paths& customPaths) const
 {
-    RetVal<io::paths> pluginPaths = fileSystem()->scanFiles(customPath, QStringList(QString::fromStdString(VST3_PACKAGE_EXTENSION)));
+    io::paths result;
+
+    for (const io::path& path : customPaths) {
+        RetVal<io::paths> paths = fileSystem()->scanFiles(path, QStringList(QString::fromStdString(VST3_PACKAGE_EXTENSION)));
```

Please, don't ignore errors